### DISCUSSION TITLE
List-All-Microsoft-Booking-Mailboxes

### DISCUSSION THREAD

**Discussion by diecknet-discussions** [diecknet/diecknet-blog#338] [Blog Discussions]
# List-All-Microsoft-Booking-Mailboxes

If you’re using Microsoft Booking in your Microsoft 365 Tenant, you might want to list all of the Booking calendars. Booking uses Exchange Online Mailboxes in the background. Every Booking Calendar has a corresponding Mailbox of the type “SchedulingMailbox”. If an user gets assigned “Administrator” for a Booking Calendar, they get “FullAccess” permissions for the Scheduling Mailbox.
List all Booking Calendars with permission I wrote a PowerShell script to list all the Booking Mailboxes with the users that have access.

https://diecknet.de/en/2021/11/12/List-All-Microsoft-Booking-Mailboxes/

**Comment by raplru** [comment]
What if I want to export all the roles of a booking page? Also Viewer, Planner, Member etc.

**Reply by diecknet** [reply]
I think you need to use the Graph API for that - not possible from the Exchange Online PowerShell.
https://devblogs.microsoft.com/microsoft365dev/using-scripts-for-microsoft-bookings-calendar-and-mailbox/
